solr_schema_init/create_isb_core_schema.py

```diff
@@ -1,4 +1,6 @@
 import json
```

Comment:
Changes here are just to update the existing schema -- you may need to fiddle with it a it to get it to match your local version.